warp10/src/main/java/io/warp10/script/mapper/FILLMAPPER.java

```diff
@@ -0,0 +1,90 @@
+//
+//   Copyright 2020  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+package io.warp10.script.mapper;
+
+import io.warp10.script.NamedWarpScriptFunction;
+import io.warp10.script.WarpScriptException;
+import io.warp10.script.WarpScriptLib;
+import io.warp10.script.WarpScriptMapperFunction;
+import io.warp10.script.WarpScriptStack;
+import io.warp10.script.WarpScriptStackFunction;
+
+/**
+ * Wrap a Mapper so empty buckets are filled by the mapper and non-empty buckets are left untouched
```

Comment:
This comment deserves a rewrite, bucket is a term used in BUCKETIZE, not so in MAP, so the comment deserves to explain more thoroughly the intent, i.e. to generate values for ticks which fall on bucket ends with no associated values when MAP is called on a bucketized GTS. The comment must enable someone diving into the code to fully understand what it does.